# Goal
string ACTION_CHANGE_ANGLE = "change_angle"
string ACTION_RESET_POSITION = "reset_position"
string action
std_msgs/Float64 angle_deg
---
# Result
bool success
string message
---
# Feedback
float32 progress